### ISSUE TITLE
Only one router and one Raspberry Pi?

### ISSUE THREAD

**Issue by boatboat123** [nexmonster/nexmon_csi_bin#11]
After setting the Raspberry Pi to listening mode, an additional device needs to be connected to the router to generate traffic. Can the Raspberry Pi be connected to a router to generate traffic and collect CSI?

**Comment by zeroby0** [comment]
No, you'll need a third device. The Pi wont be connected to the router wirelessly, it'll be in listening only mode. So you'll need a second device that is connected to the router and generate traffic. Connecting a wireless dongle to the Pi wouldn't work -- we'll be removing wpa_supplicant so you can't connect the pi wirelessly at all.

**Comment by boatboat123** [comment]
If I don't use the third device and instead have the router broadcast or send traffic to a defined destination IP (which may not exist), can the Raspberry Pi correctly monitor the traffic that the router sends out?
